simulation_parameters/microbe_stage/membranes.json

```diff
@@ -84,8 +84,8 @@
     "MovementWigglyness": 0.5,
     "OsmoregulationFactor": 0.6,
     "ResourceAbsorptionFactor": 0.6,
-    "Hitpoints": 140,
-    "PhysicalResistance": 0.6,
+    "Hitpoints": 160,
+    "PhysicalResistance": 1.2,
```

Comment:
I suppose this is a similar change. Though I'm not sure we want to make it so that these membranes are just flat out better, instead of being tradeoffs between either toxin or physical resistance.